core/drivers/sub.mk

```diff
@@ -2,3 +2,4 @@ srcs-$(CFG_PL011) += pl011.c
 srcs-$(CFG_GIC) += gic.c
 srcs-$(CFG_SUNXI_UART) += sunxi_uart.c
 srcs-$(CFG_8250_UART) += serial8250_uart.c
+srcs-$(CFG_LS_UART) += ns16550.c
```

Comment:
`CFG_16550_UART` would be better because this is not related to the LS platform but to the 16550 chip